dask_ml/model_selection/_incremental.py

```diff
@@ -654,13 +655,15 @@ def _get_params(self):
             return ParameterSampler(self.parameters, self.n_initial_parameters)
 
     def _additional_calls(self, info):
-        best = self._adapt(info)
-        bad = set(info) - set(best)
+        self._all_models.update(set(info.keys()))
+        instructions = self._adapt(info)
+        bad = set(self._all_models) - set(instructions)
 
         # Keep models in case they quickly reach a high plateau (yes, unlikey)
-        best.update({model_id: 0 for model_id in bad})
+        instructions.update({model_id: 0 for model_id in bad})
+
         out = {}
-        for k, steps in best.items():
+        for k, steps in instructions.items():
             records = info[k]
             if self.max_iter and len(records) >= self.max_iter:
                 out[k] = 0
```

Comment:
TODO: resolve the selection of the beginning of the plateau. Right now, the code assumes every records corresponds to 1 partial fit call. This may not be true if adapt is general. Case in point: `old = records[-self.patience]["score"]`.